controllers/application_controller.go

```diff
@@ -226,27 +224,24 @@ func (r *ApplicationReconciler) Reconcile(ctx context.Context, req ctrl.Request)
 }
 
 // SetupWithManager sets up the controller with the Manager.
-func (r *ApplicationReconciler) SetupWithManager(mgr ctrl.Manager) error {
+func (r *ApplicationReconciler) SetupWithManager(ctx context.Context, mgr ctrl.Manager) error {
 	gofakeit.New(0)
-	opts := zap.Options{
-		TimeEncoder: zapcore.ISO8601TimeEncoder,
```

Comment:
does that mean timestamp will no longer be in UTC?

Reply:
it is. the logger will be from ctx created in `main.go`. and main.go has the timestamp set in UTC.
https://github.com/redhat-appstudio/application-service/blob/c4c1117547245c27bf5282a87b7e189891e0e721/main.go#L84-L93